File: Selenium/Project/Activity5.java
package groupProject;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import java.time.Duration;

public class Activity5 {

        public static void main(String[] args) {
            // Set up Firefox driver
            WebDriverManager.firefoxdriver().setup();
            // Create a new instance of the Firefox driver
            WebDriver driver = new FirefoxDriver();
            // Open the page
            driver.get("https://alchemy.hguy.co/jobs/");
            driver.findElement(By.linkText("Jobs")).click();
            // Print the title of the page
            WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
            String title = driver.getTitle();
            System.out.println("Jobs page title: " + title);
            Assert.assertEquals(title, "Jobs – Alchemy Jobs");
           String message = driver.findElement(By.xpath("//h1[@class='entry-title']")).getText();
            System.out.println("Login title: " + message);
            Assert.assertEquals(message, "Jobs");
            // Close the browser
            driver.close();
        }
    }

File: Selenium/Project/Activity5.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/jobs/")
# Find the "Jobs" button on the page using xpath and click it
    driver.find_element(By.LINK_TEXT, "Jobs").click()
# Print the title of the new page
    pageTitle = driver.title
    print("The page title is: ", pageTitle)
    wait = WebDriverWait(driver, 10)
    assert (pageTitle == 'Jobs – Alchemy Jobs')
    message = driver.find_element(By.XPATH, "//h1[@class='entry-title']").text
    print(message)
    assert(message == 'Jobs')



